deepcell/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.4.0"
```

Comment:
Can we just put this in the base __init__ file? I'm not sure what is best for declaring package versions.